homeassistant/package_constraints.txt

```diff
@@ -26,7 +26,7 @@ ciso8601==2.3.1
 cryptography==43.0.1
 dbus-fast==2.24.3
 fnv-hash-fast==1.0.2
-go2rtc-client==0.0.1b0
+go2rtc-client==0.0.1b1
```

Comment:
Why do we have this in package_constraints at all?

Reply:
Good question. Was added with https://github.com/home-assistant/core/pull/129144
@emontnemery Why did you add it to the constraint file?

Reply:
In #129144, hassfest added it because go2rtc became a dependency of core_config.-
go2rtc was already needed by `core_config` so it should have been in package_constraints.txt from the start to prevent  integrations from overriding it